### ISSUE TITLE
Project/Customer manager access level

### ISSUE THREAD

**Issue by damsleth** [Puzzlepart/did#1087]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
Per [this thread](https://teams.microsoft.com/l/message/19:876c3ec2e97e4186b71cbd01f5002963@thread.tacv2/1674461331778?tenantId=dcb89c10-67b8-48cc-8819-85de37e8af91&parentMessageId=1674461331778&groupId=46adb6c8-8c09-409f-a333-2d131ab6aff8) - several users need a report of total hours for a given project, without requiring the full rights of an invoice manager.

### Description
When creating or editing a project, we could include a "project managers" / "billing responsible" (or similarly named) multi user field.  
These users would be allowed to list the "total hours" report for that project.  
If the user is neither at access level >=invoice manager OR in the project billing responsible list, they'd get a message that they don't have access, (and potentially get pointed to a user who is responsible for that particular project).

This user field has uses beyond that of the "Total hours" report, e.g a future "my projects" portfolio or similar, and  gives us granularity per project, which has been missing for a while.

Note: the access level could and maybe should be at the customer level instead of the project level, but I can see uses cases for both.
